builder/builder/Config.cs

```diff
@@ -6,7 +6,7 @@ namespace builder
     public static class Config
     {
         public static readonly Version Version = 
-            new StableVersion(1, 83, 0);
+            new StableVersion(1, 84, 0);
```

Comment:
@sergey-shandar boost 1.84.0 is available for windows.

Reply:
@sergey-shandar boost 1.85.0 is available

Reply:
I will release 1.85 in a few days.

Reply:
@sergey-shandar Consider update of nuget commandline paket to at least 5.11.6 as 5.11.5 has a security warning, see https://github.com/chcg/getboost/security/dependabot/5 .